package/deb_control

```diff
@@ -0,0 +1,7 @@
+Package: victoria-metrics
```

Comment:
I'd move deb-related files to `/package/deb` subdirectory.

Reply:
Done, .deb specific files are now under `package/deb`.